src/utilities/radius_expansion_method_with_servo_control_run.py

```diff
@@ -163,9 +165,43 @@ def GetMaximumVelocity(self):
             if velocity_magnitude > max_particle_velocity:
                 max_particle_velocity = velocity_magnitude
         return max_particle_velocity
-    
+
```

Comment:
The modifications done in this file should also be done in the "src/utilities/radius_expansion_method_with_servo_control_run_final.py", otherwise there be some errors in some cases

Reply:
They are different in some parts. As I call it "-final.py", it will be called in the last try value of this list target_packing_density_list = [target_packing_density-0.02, target_packing_density-0.05, target_packing_density-0.075, target_packing_density-0.1, target_packing_density-0.15].

Reply:
Maybe we can collapse both scripts in only one